src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1527,8 +1527,8 @@
   <Node Name="BoundConvertedTupleLiteral" Base="BoundTupleExpression">
     <!-- Converted tuple must have a type -->
     <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
-    <!-- Natural type is preserved for the purpose of semantic model. Can be null -->
-    <Field Name="NaturalTypeOpt" Type="TypeSymbol" Null="allow"/>
+    <!-- Original tuple is preserved for the purpose of semantic model -->
+    <Field Name="SourceTuple" Type="BoundTupleLiteral" SkipInVisitor="ExceptNullabilityRewriter" />
```

Comment:
>SourceTuple [](start = 17, length = 11)

I will have to make this field optional when merging, as nodes of `BoundConvertedTupleLiteral` need to be created when lowering tuple conversions and tuple equality.  We currently generate `BoundTupleLiteral`, but that isn't correct as such nodes should be gone by the end of initial binding and we don't want to have to deal with both forms.